src/app/ngrx/effects/castle.effects.spec.ts

```diff
@@ -443,17 +452,18 @@ describe('CastleEffects', () => {
     const success = new ACTIONS.CastlePodcastRanksSuccessAction({
       id: podcast.id,
       group: GROUPTYPE_AGENTNAME,
+      filter: undefined,
       interval: INTERVAL_DAILY,
       ranks: podcastAgentNameRanks,
       downloads: podcastAgentNameDownloads
     });
 
-    actions$.stream = hot('-a', { a: action });
-    const expected = cold('-r', { r: success });
+    actions$.stream = hot('-ab', { a: action, b: action });
```

Comment:
I changed these to mergeMap so I could load both geocountry and geosubdiv ranks requests at the same time